src/transformStateWithClones.js

```diff
@@ -8,6 +8,40 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const arrayWithObjects = [];
```

Comment:
Use a more descriptive name for the array that will store the state history. For example, 'stateHistory' would be more informative than 'arrayWithObjects'.